### PR TITLE
feat(menubar): expose menubar styling to beautiful

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

File: lib/menubar/init.lua [hunk 1]
```diff
@@ -39,6 +39,33 @@ local function get_screen(s)
...
+local border_color = theme.menubar_border_color or theme.menu_border_color
+
+--- Menubar selected item text color.
+-- @beautiful beautiful.menubar_fg_normal
+local focused_fg_color = theme.menubar_fg_focus or theme.menu_fg_focus or theme.fg_focus
+
+--- Menubar selected item background color.
+-- @beautiful beautiful.menubar_bg_normal
+local focused_bg_color = theme.menubar_bg_focus or theme.menu_bg_focus or theme.bg_focus
+
```
Comment (Elv13) [inline lib/menubar/init.lua]: I guess this should be done within the function. I there is at least 1 person who try to implement changing from dark theme to light theme at runtime without restarting Awesome.

And yes, there is a lot of broken things related to dynamic theme changes, but lets try not to add more.

Comment (actionless) [reply lib/menubar/init.lua]: done

Comment (Elv13) [reply lib/menubar/init.lua]: thanks

Comment (actionless) [reply lib/menubar/init.lua]: your last review got dismissed

**Comment by Elv13** [comment]
Sorry, I haven't been paying attention of the PR stuck due to codecov, sorry for the merge delay

**Pull request merged**